network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/CalculatedBus.java

```diff
@@ -147,6 +148,40 @@ public Bus setAngle(double angle) {
         return this;
     }
 
+    @Override
+    public double getFictitiousP0() {
+        return voltageLevelResource.getAttributes().getNodeToFictitiousP0().values().stream().reduce(0.0, Double::sum);
```

Comment:
voltageLevelResource.getAttributes().getNodeToFictitiousP0() can be null, I think ...

Reply:
There is still no equivalent here of what is done in powsybl-core with Networks.getNodes static method to retrieve only the list of nodes corresponding to this calculated bus ... (here, you sum all the map values)